src/server/utils/onModuleLoad.js

```diff
@@ -150,21 +157,19 @@ export default function onModuleLoad({
 
     Object.entries(requiredExternals).forEach(([externalName, requestedExternalVersion]) => {
       const providedExternal = RootModule[CONFIGURATION_KEY].providedExternals[externalName];
-
+      // assuming fallbacks available for now
       if (!providedExternal) {
         messages.push(`External '${externalName}' is required by ${moduleName}, but is not provided by the root module`);
+        registerExternalFallback({ moduleName, externalName, requestedExternalVersion });
       } else if (!semver.satisfies(providedExternal.version, requestedExternalVersion)) {
         const failedExternalMessage = `${externalName}@${requestedExternalVersion} is required by ${moduleName}, but the root module provides ${providedExternal.version}`;
-        if (process.env.ONE_DANGEROUSLY_ACCEPT_BREAKING_EXTERNALS) {
-          console.warn(failedExternalMessage);
-        } else {
-          messages.push(failedExternalMessage);
-        }
+        registerExternalFallback({ moduleName, externalName, requestedExternalVersion });
```

Comment:
I guess the module will call a function for that particular external (e.g. `module.exports = something()` that will try to find the module's externals in the Map, if it isn't available then go to the Root Module and look for it, and, if can't still find it them reject the module